db/migrate/20170619105503_remove_workflows.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveWorkflows < ActiveRecord::Migration[5.0]
+  def change
+    rename_column :workflows, :workflowable_id, :detainee_id
+    remove_column :workflows, :type
```

Comment:
It's a good practice to specify the `type` of the field so that the migrations know how to rollback to the initial state.